homework-14.js

```diff
@@ -0,0 +1,22 @@
+//Реалізуйте функцію generateKey(length, characters), яка повертає рядок випадкових символів із набору characters довжиною length
+
+
+
+function generateKey(length, characters) {
+    let arrayCharacters = Array.from(characters);
+    let newArray = [];
+
+    for (let k = 0; k < length; k++) {
+        let min = 0;
+        let randomIndex;
+        randomIndex = Math.floor(Math.random() * (arrayCharacters.length - min + 1) ) + min;
+        newArray.push(arrayCharacters[randomIndex]);
```

Comment:
можна було б трохи спростити

```suggestion
        let randomIndex = Math.floor(Math.random() * (arrayCharacters.length + 1) );
        newArray.push(arrayCharacters[randomIndex]);
```